src/plugins/cosmos/components/modals/StakingConfirm/views/Broadcast.tsx

```diff
@@ -0,0 +1,173 @@
+import { InfoOutlineIcon } from '@chakra-ui/icons'
+import { Flex } from '@chakra-ui/layout'
+import { Button, Link, ModalHeader, Text as CText, Tooltip } from '@chakra-ui/react'
+import { CAIP19 } from '@shapeshiftoss/caip'
+import { chainAdapters } from '@shapeshiftoss/types'
+import { Asset } from '@shapeshiftoss/types'
+import { AprTag } from 'plugins/cosmos/components/AprTag/AprTag'
+import { FormProvider, useForm } from 'react-hook-form'
+import { useTranslate } from 'react-polyglot'
+import { Amount } from 'components/Amount/Amount'
+import { MiddleEllipsis } from 'components/MiddleEllipsis/MiddleEllipsis'
+import { Text } from 'components/Text'
+import { useModal } from 'context/ModalProvider/ModalProvider'
+import { BigNumber } from 'lib/bignumber/bignumber'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+
+export enum InputType {
+  Crypto = 'crypto',
+  Fiat = 'fiat'
+}
+
+export enum Field {
+  FeeType = 'feeType'
+}
+
+export type StakingValues = {
+  [Field.FeeType]: chainAdapters.FeeDataKey
+}
+
+type StakeProps = {
+  assetId: CAIP19
+  apr: string
+  fiatRate: BigNumber
+  cryptoStakeAmount: BigNumber
+}
+
+// TODO: Make this a derived selector after this is wired up
+function calculateYearlyYield(apy: string, amount: string = '') {
+  return bnOrZero(amount).times(apy).toString()
+}
+
+const DEFAULT_VALIDATOR_NAME = 'Shapeshift Validator'
+
+// TODO: Wire up the whole component with staked data
+export const Broadcast = ({ apr, assetId, cryptoStakeAmount, fiatRate }: StakeProps) => {
+  const methods = useForm<StakingValues>({
+    defaultValues: {
+      [Field.FeeType]: chainAdapters.FeeDataKey.Average
+    }
+  })
+
+  const { handleSubmit } = methods
+
+  const { cosmosStakingConfirm } = useModal()
+
+  const onSubmit = (_: any) => {
+    // TODO: handle submit when wired up
+  }
+
+  const cryptoYield = calculateYearlyYield(apr, cryptoStakeAmount.toPrecision())
+  const fiatYield = bnOrZero(cryptoYield).times(fiatRate).toPrecision()
+
+  const translate = useTranslate()
+
+  const handleCancel = () => {
+    cosmosStakingConfirm.close()
+  }
+
+  // TODO: wire me up, parentheses are nice but let's get asset name from selectAssetNameById instead of this
+  const asset = (_ => ({
+    name: 'Osmosis',
+    symbol: 'OSMO',
+    caip19: assetId,
+    chain: 'osmosis'
+  }))(assetId) as Asset
+  const txDetails = {
+    explorerTxLink: 'https://etherscan.io/tx/',
+    tx: {
+      txid: '42foobar42'
+    }
+  }
+  return (
+    <FormProvider {...methods}>
+      <Flex
+        as='form'
+        pt='14px'
+        pb='18px'
+        px='30px'
+        onSubmit={handleSubmit(onSubmit)}
+        flexDirection='column'
+        alignItems='center'
```

Comment:
It should be tested, but I think that if you remove this `alignItems` you can remove every `width='100%'` inside this `Flex`

Because every Flex, as `display: flex`, will have a natural width of `100%`